### DISCUSSION TITLE
Invalid value *IDW_BlockReferenceLayer for attribute "layer" in entity SEQEND(#None).

### DISCUSSION THREAD

**Discussion by jitendra12300** [mozman/ezdxf#995] [Issues]
I am getting the below error when reading (iam) converted DXF files.

**Code :** 
import ezdxf
def read_text_attributes(filePath):
    # Replace 'your_file.dxf' with the path to your DXF file
    dxf_file_path = filePath
    # Load the DXF file
    doc = ezdxf.readfile(dxf_file_path)# **Error Coming in this line**
    # Access the modelspace (where most of the entities are typically located)
    msp = doc.modelspace()

**Error:** 
Invalid value *IDW_BlockReferenceLayer for attribute "layer" in entity SEQEND(#None).

**Comment by mozman** [comment]
The `*` is an invalid character for a table name like the layer table.

**Comment by mozman** [comment]
AutoCAD seems to accept the `*` as first letter of a layer name and only as the first letter e.g. `Layer*` is invalid. 

This will be fixed in the next release.

**Reply by jitendra12300** [reply]
Thank you, could you please let me know, when we can expect the next release?

**Reply by jitendra12300** [reply]
![Assembly Layer](https://github.com/mozman/ezdxf/assets/122965223/4be06999-b5b1-4d46-a5a3-279477356c65)

**Comment by jitendra12300** [comment]
@mozman 
Please reply, I am waiting for your answer.

**Comment by jitendra12300** [comment]
Any update on this
